packages/core/src/electron-browser/menu/electron-main-menu-factory.ts

```diff
@@ -14,6 +14,8 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
+// tslint:disable:no-any
```

Comment:
[minor]: I would prefer having three distinct `no-any` instead of disabling this rule for the entire file.